src/ngraph/op/broadcast.cpp

```diff
@@ -65,7 +66,7 @@ std::pair<bool, AxisSet> op::v1::Broadcast::get_broadcast_axes() const
             auto target_shape = get_input_shape(1);
             NGRAPH_CHECK(target_shape.size() == 1);
             auto axes_mapping_val =
-                static_pointer_cast<op::Constant>(input_value(2).get_node_shared_ptr())
+                static_pointer_cast<op::v0::Constant>(input_value(2).get_node_shared_ptr())
```

Comment:
If there were an `op::v3::Constant`, `is_constant()` would return `true` and `axes_mapping_val` would be garbage (`static_pointer_cast` means you know you're correct). Use `as_type_ptr<op::v0::Constant>(get_input_node_shared_ptr(2))` and then that won't happen. You should be able to replace the `is_constant()` with a little reorganizing.

Reply:
Done